packages/main/src/contacts.ts

```diff
@@ -36,8 +35,19 @@ export default class ContactsApiClient {
 
   async downloadContacts(accountId: ID) {
     const connections = await this.getContacts(accountId)
-    if (connections) {
-      persistContacts(accountId, connections)
+    for (const connection of connections || []) {
+      for (const email of connection.emailAddresses || []) {
```

Comment:
I think that `for ... of` is more idiomatic than `.forEach()`. It used to be that there was no great way to iterate over arrays, and `.forEach()` was the way to go. But since `for ... of` was added in ES6 I prefer to use it. An important advantage is that while `.forEach()` only works on arrays (or other values that implement a method with the same name) `for ... of` can iterate over any iterable value: arrays, argument lists, DOM node lists, Immutable.js collections, etc.